prow/plugins/jira-linker/jira-linker.go

```diff
@@ -110,6 +111,20 @@ func handle(gc githubClient, log *logrus.Entry, config plugins.JiraLinker, event
 	return nil
 }
 
+func findJiraUrl(config plugins.JiraLinker, repo string) string {
+	jiraServerURL := config.JiraBaseUrl
+out:
```

Comment:
Since this is a separate func now, you actually don't need the break/label structure anymore - you can just `return` from inside the loop as soon as you find a match.